### PR TITLE
Animate close button on tabs (fix #802)

### LINKED ISSUE #802: Animate close button on tabs
Follow up for https://github.com/mozilla/browser.html/issues/789.

### PR DESCRIPTION
**Pull request by joshua-s**
See #802

### CODE REVIEW

File: src/browser/sidebar/tab.js [hunk 1]
```diff
@@ -98,7 +98,9 @@ const styleSheet = StyleSheet.create({
     position: 'absolute',
     paddingLeft: '10px',
     right: 0,
-    top: 0
+    top: 0,
+    transition: `transform 400ms cubic-bezier(0.215, 0.610, 0.355, 1.000),
+        opacity 300ms ease-out`
```
Comment (gordonbrander) [inline src/browser/sidebar/tab.js]: nit: the spacing should line up with `transform`, or else maybe be on the same line.

Comment (gordonbrander) [inline src/browser/sidebar/tab.js]: nit: we should use normal single-quote marks here, since we're not using a variable

**Comment by gordonbrander** [comment]
Hey, this looks great. Just a couple of nits (noted). You can just push fixes to this branch as a follow-up commit. No need to rebase. I'll test this code tomorrow am.

**Comment by joshua-s** [comment]
Notes addressed!

**Comment by gordonbrander** [comment]
Awesome! Works great!

**Pull request merged**